### PR TITLE
mount new persistent data directory for analysis results & fixup Docker builds

### PR DESCRIPTION
**Pull request by profjsb**
* Update `.dockerignore` so we do not send more data into a new docker image than is needed
* Update `Dockerfile` to point to the newly mounted `/skyportal/persistentdata` directory
* Update `config.yaml.defaults` for the analysis data to be written to  `/skyportal/persistentdata`
* Make sure that `docker-compose.yaml` is up to date.

To make SP docker images
```bash
make docker-local
```
(On ARM architectures on Mac you may need to run `export DOCKER_DEFAULT_PLATFORM=linux/amd64` first)

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -7,19 +7,22 @@ services:
       - "9000:5000"
       - "9001:5001"
     volumes:
-      - thumbnails:/skyportal/static/thumbnails
+      - thumbnails:/skyportal/static/thumbnails:z
```
Comment (mcoughlin) [inline docker-compose.yaml:10]: Maybe add a content that :z is relevant for all shared content?

**Review by mcoughlin — APPROVED** [review]
Looks like nice quality of life improvements here.

**Comment by mcoughlin** [comment]
@Theodlz Flagging this for you given your attachments PR.

File: docker-compose.yaml (hunk 1 above)
Comment (profjsb) [reply docker-compose.yaml:13]: done

**Pull request merged**